src/main/java/racingCar/Car.java

```diff
@@ -2,49 +2,40 @@
 
 public class Car {
     private static final int LIMIT_NUM = 4;
-    private static final int LIMIT_NAME_LENGH = 5;
-    private int distance;
-    private String name;
+    private Distance distance;
+    private Name name;
```

Comment:
```suggestion
    private final Name name;
```
인스턴스 변수는 가능하면 final로 구현